src/lib.rs

```diff
@@ -142,19 +157,17 @@ fn level_value(level: Option<Level>) -> i8 {
     }
 }
 
-fn level_enum(verbosity: i8) -> Option<Level> {
+fn level_enum(verbosity: i16) -> Option<Level> {
```

Comment:
> The internal representation in calculations has been changed from i8 to
i16 to prevent overflows when performing calculations using u8 values.


Please make this its own commit

Reply:
#115